djaoapp/templates/saas/_subscriber_sidebar.html

```diff
@@ -17,7 +17,7 @@
 {% if urls.organization.roles %}
 {% for role_title, role_url in urls.organization.roles.items() %}
 <li{% if role_url == request.path %} class="active"{% endif %}>
-  <a href="{{role_url}}"><i class="fa fa-users"></i><span class="">{{role_title|pluralize}}</span></a>
+  <a href="{{role_url}}"><i class="fa fa-users"></i><span class="">{{role_title}}s</span></a>
```

Comment:
That one might be tricky even with a `title_plural` property on `RoleDescription`.